app/src/main/res/values/colors.xml

```diff
@@ -3,6 +3,7 @@
 	<color name="colorPrimaryDark">#476693</color>
 	<color name="blue">#4a90e2</color>
     <color name="fadedblue"> #994a90e2</color>
+	<color name="lightblue">#b7d3f3</color>
```

Comment:
can you fix the indentation in the last color, and fix the casing so it's consistent? `colorPrimaryDark` is camelcased, but nothing else seems to be?